packages/hash/frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx

```diff
@@ -21,7 +21,9 @@ export const PageSidebar: FunctionComponent = () => {
   const router = useRouter();
   const { sidebarOpen, closeSidebar } = useSidebarContext();
   const { activeWorkspaceAccountId } = useContext(WorkspaceContext);
-  const { pageEntityId } = useRoutePageInfo({ allowUndefined: true }) ?? {};
+  /** @todo: re-use the `pageEntityId` provided by `getServerSideProps` */
+  const { routePageEntityId } =
+    useRoutePageInfo({ allowUndefined: true }) ?? {};
```

Comment:
I wasn't sure what the best way of avoiding this would be, as the sidebar is not rendered as a child of the `[page-slug].page.tsx` page.

Reply:
Discussed this internally, and came to the reaslisation that the `pageEntityUuid` could be used instead. Have addressed in https://github.com/hashintel/hash/pull/1650/commits/48aab54f8e6b3b5e04845cdb1e70dbd731ac0bf7